src/Graphics/src/Graphics.Skia/PublicAPI/net-android/PublicAPI.Shipped.txt

```diff
@@ -1 +1,3 @@
 #nullable enable
+Microsoft.Maui.Graphics.Skia.Resource
+Microsoft.Maui.Graphics.Skia.Resource.Resource() -> void
```

Comment:
APIs like this, we are working on a setting, so we can prevent them from being generated:

https://github.com/xamarin/xamarin-android/pull/7721

So in a future PR, we will revert several of the changes in `PublicAPI.Shipped.txt`.